Tools/conversation_manager.py

```diff
@@ -18,29 +21,45 @@ def __new__(cls):
             cls._instance = super(ConversationManager, cls).__new__(cls)
         return cls._instance
 
+    @classmethod
+    def get_user_conversation(cls, telegram_user_id: int) -> Tuple['ConversationManager', str]:
```

Comment:
this method does the same what get_conversation_history.
telegram_user_id should be converted to user_id from PostgreSQL before calling get_conversation_history.
Adding cache logic to this module is making mess.
Method can be removed.